pkgs/development/python-modules/matplotlib/default.nix

```diff
@@ -4,6 +4,7 @@
 , enableGhostscript ? true, ghostscript ? null, gtk3
 , enableGtk2 ? false, pygtk ? null, gobjectIntrospection
 , enableGtk3 ? false, cairo
+# default backend, enabled in all-packages.nix except on darwin
```

Comment:
Why not just put `enableTk ? !stdenv.isDarwin` in both `matplotlib/default.nix` and `matplotlib/2.nix`? You end up cross referencing twice anyway, once in nix at `top-level/python-packages.nix` and once in english in this comment (not to mention this misses the cross reference comment for `matplotlib/2.nix`).

Reply:
Because I assumed that you couldn't access the contents of the attribute list within the attribute list. But apparently you can! Changed.